src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -25,4 +24,40 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
-}
+
+    @GetMapping
+    public ResponseEntity getLines() {
+        LinesResponse lines = lineService.getLines();
+        return ResponseEntity.ok(lines.getLines());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity getLine(@PathVariable("id") Long id) {
```

Comment:
```suggestion
    public ResponseEntity<LineResponse> getLine(@PathVariable("id") Long id) {
```

제네릭에서 타입을 명시하지 않으면 어떠한 문제가 있을지 고민해보아요

Reply:
ResponseEntity 타입이 제네릭인 줄 몰랐네요 .. 좋은 지적 감사합니다. 
타입을 명시하지 않으면 return되는 Entity 타입이 기대하지 않는 값이 오더라도 알 수가 없는것 같습니다. 맞나요? ^^;;